building/devices/stm32mp1.rst

```diff
@@ -62,10 +62,10 @@ A usual short fecth/build/load shell sequence is like the one below:
 Command ``sgdisk -e`` fixes the GPT backup data which location depends on
 storage device effective size.
 
-.. _STM32MP135F-DK1: https://www.st.com/en/evaluation-tools/stm32mp135x-dk.html
+.. _STM32MP135F-DK: https://www.st.com/en/evaluation-tools/stm32mp135f-dk.html
 .. _STM32MP157A-DK1: https://www.st.com/en/evaluation-tools/stm32mp157a-dk1.html
 .. _STM32MP157C-DK2: https://www.st.com/en/evaluation-tools/stm32mp157c-dk2.html
 .. _STM32MP157C-EV1: https://www.st.com/en/evaluation-tools/stm32mp157c-ev1.html
-.. _Wiki STM32MP135X-DK: https://wiki.st.com/stm32mpu/wiki/STM32MP135x-DK_-_hardware_description
 .. _Wiki STM32MP157X-DKX: https://wiki.st.com/stm32mpu/wiki/STM32MP157X-DKX_-_hardware_description
 .. _Wiki STM32MP157C-EV1: https://wiki.st.com/stm32mpu/wiki/STM32MP157C-EV1_-_hardware_description
+.. _Wiki STM32MP135F-DK: https://wiki.st.com/stm32mpu/wiki/STM32MP135x-DK_-_hardware_description
```

Comment:
As wiki is pointing at 135X-DK, I'd stay as: _Wiki STM32MP135X-DK. Also can be put back above 
_Wiki STM32MP157X-DKX